components/tools/OmeroJava/test/integration/ProjectionServiceTest.java

```diff
@@ -235,6 +235,19 @@ public void testSameTimepointInterval() throws Exception {
     }
 
     /**
+     * Test the projection with a restricted but valid timepoint range.
+     *
+     * @throws Exception Thrown if an error occurred.
+     */
+    @Test
+    public void testTimepointIntervalNotFullRange() throws Exception {
+        Pixels pixels = importImage();
+        List<Integer> channels = Arrays.asList(0);
+        projectImage(pixels, 2, 4, 1, 3, 1,
```

Comment:
Feel free, I tested it locally. More importantly, the test has an assert for the number of t points in the new image: https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroJava/test/integration/ProjectionServiceTest.java#L121